### PR TITLE
Minor improvements to messages view

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

File: go/conversation/tests.py [hunk 1]
```diff
@@ -786,11 +786,12 @@ def test_message_list_statistics(self):
...
 
-    def test_message_list_inbound_uniques_display(self):
-        conv = self.user_helper.create_conversation(u'dummy', started=True)
-        self.msg_helper.add_inbound_to_conv(conv, 10)
+    def test_message_list_title_display(self):
+        conv = self.user_helper.create_conversation(
+            u'dummy', name=u'Foo', started=True)
         response = self.client.get(self.get_view_url(conv, 'message_list'))
-        self.assertContains(response, 'Messages from 10 unique people')
+        print response
```
Comment (hodgestar) [inline go/conversation/tests.py]: Stray debug print.

**Comment by hodgestar** [comment]
One stray debug print. Otherwise looks good.

**Comment by justinvdm** [comment]
Ready for re-review.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**